### ISSUE TITLE
Panics with error `NotFound`

### ISSUE THREAD

**Issue by farzadmf** [lusingander/stu#27]
Hi, I installed the `brew` version on my Mac, logged in to my AWS account, but I get an error:

```
❯ RUST_BACKTRACE=1 stu
thread 'tokio-runtime-worker' panicked at src/cache.rs:32:32:
called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }
stack backtrace:
   0: _rust_begin_unwind
   1: core::panicking::panic_fmt
   2: core::result::unwrap_failed
   3: stu::cache::SimpleStringCache::new
   4: stu::run::{{closure}}::{{closure}}
   5: tokio::runtime::task::core::Core<T,S>::poll
   6: tokio::runtime::task::raw::poll
   7: tokio::runtime::scheduler::multi_thread::worker::Context::run_task
   8: tokio::runtime::scheduler::multi_thread::worker::Context::run
   9: tokio::runtime::context::scoped::Scoped<T>::set
  10: tokio::runtime::context::runtime::enter_runtime
  11: tokio::runtime::scheduler::multi_thread::worker::run
  12: <tokio::runtime::blocking::task::BlockingTask<T> as core::future::future::Future>::poll
  13: tokio::runtime::task::core::Core<T,S>::poll
  14: tokio::runtime::task::harness::Harness<T,S>::poll
  15: tokio::runtime::blocking::pool::Inner::run
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Comment by lusingander** [comment]
Ah, thank you, this is a regression in 0.6.0.

**Comment by lusingander** [comment]
In v0.6.0, the process was not performed correctly when starting stu for the first time. I've released v0.6.1 with the fix.

Thank you very much for reporting this.

**Comment by farzadmf** [comment]
Thank you @lusingander for the quick action.

Any updates on #26 (making `brew` installation available on Linux as well)?
